src/main/java/com/gtnewhorizons/angelica/mixins/late/client/minefactoryreloaded/MixinRedNetCableRenderer.java

```diff
@@ -0,0 +1,41 @@
+package com.gtnewhorizons.angelica.mixins.late.client.minefactoryreloaded;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.renderer.RenderBlocks;
+import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;
+import net.minecraft.world.IBlockAccess;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+import powercrystals.minefactoryreloaded.render.block.RedNetCableRenderer;
+import powercrystals.minefactoryreloaded.setup.MFRConfig;
+import powercrystals.minefactoryreloaded.tile.rednet.TileEntityRedNetCable;
+
+@Mixin(value = RedNetCableRenderer.class, remap = false)
+public abstract class MixinRedNetCableRenderer extends TileEntitySpecialRenderer {
+
+    @Shadow
+    private boolean renderCable(IBlockAccess world, int x, int y, int z, int brightness, TileEntityRedNetCable _cable) {
+        throw new AbstractMethodError("Shadow");
+    }
+
+    /**
+     * @author JL2210
+     * @reason Thread safety compat (check getTileEntity for null)
+     */
+    @Overwrite
```

Comment:
I tend to dislike overwrite unless there's no alternative.  

I'd see if there's a way to redirect the _cable.onRender() - or wrap it using MixinExtras to check for null

Reply:
I rewrote it as an injector, please take a look; I think I did it right but I'm pretty new to mixins. It works, at least.